src/aws/cloudwatch-sink.cpp

```diff
@@ -363,16 +370,19 @@ void CloudWatchSink::init_context_info(nlohmann::json& dst,
             throw std::runtime_error(fmt::format("Wrong context_info destination type {}", dst.type_name()));
     }
 
-    for (auto const& itr : context_info)
+    for (auto const& ci_itr : {log.context_info(), context_info, global_context_info})
     {
-        if (!dst.contains(itr.first))
+        for (auto const& itr : ci_itr)
         {
-            dst[itr.first.data()] = itr.second;
+            if (!dst.contains(itr.first))
```

Comment:
@aradyaron - if I understand correctly, this means that the order of precedence for context keys is:
1. log_context
2. logger_context
3. global_context

i.e if there are conflicting keys then the most local context triumphs.
This make sense to me, but I just want to make sure that this is indeed by design?

Reply:
Made this explicit